include/tdnferror.h

```diff
@@ -175,6 +175,17 @@ extern "C" {
 //FIPS error
 #define ERROR_TDNF_FIPS_MODE_FORBIDDEN       2600
 
+//METALINK error
+#define ERROR_TDNF_MLK_PARSER_INVALID_DOC_OBJECT       2700
```

Comment:
pls add description for the errors in TDNF_ERROR_TABLE  in client/defines.h

Reply:
done